src/Magnum/OvrIntegration/Hmd.h

```diff
@@ -358,24 +370,110 @@ class MAGNUM_OVRINTEGRATION_EXPORT Hmd {
          */
         void setDebugHudStereoMode(DebugHudStereoMode mode) const;
 
+        /**
+         * @brief Set layer HUD mode
+         *
+         * Layer HUD enables the HMD user to see information about a layer.
+         */
+        void setLayerHudMode(LayerHudMode mode) const;
+
         /** @brief Tracking state */
         StatusFlags trackingState() const {
             return {StatusFlag(_trackingState.StatusFlags)};
         }
 
+        /** @brief Name of the currently active Oculus profile */
+        std::string user() const {
+            return ovr_GetString(_session, OVR_KEY_USER, "");
+        }
+
+        /** @brief Name set in the currently active Oculus profile */
+        std::string playerName() const {
+            return ovr_GetString(_session, OVR_KEY_NAME, "");
+        }
+
+        /** @brief Gender set in the currently active Oculus profile */
+        std::string playerGender() const {
+            return ovr_GetString(_session, OVR_KEY_GENDER, OVR_DEFAULT_GENDER);
+        }
+
+        /** @brief Player height set in the currently active Oculus profile */
+        Float playerHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_PLAYER_HEIGHT);
+        }
+
+        /** @brief Eye height set in the currently active Oculus profile */
+        Float eyeHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_EYE_HEIGHT);
+        }
+
+        /**
+         * @brief Inter populary distance set in the currently active Oculus
+         * profile
```

Comment:
As noted previously, we have to put `@brief`s onto single line to make Doxygen parse them properly. Also, "interpupillary", I think :)